src/inc/request-handler.ts

```diff
@@ -1,35 +1,45 @@
-import type { Auth, HTTPRequest } from '../'
+import type { AxiosRequestConfig } from 'axios'
+import type { NuxtAxiosInstance } from '@nuxtjs/axios'
+import type { Scheme } from '../'
 import ExpiredAuthSessionError from './expired-auth-session-error'
 
 export default class RequestHandler {
-  constructor (public $auth: Auth) {}
+  public scheme: Scheme
+  public $axios: NuxtAxiosInstance
+  public interceptor: any
+
+  constructor (scheme, axios) {
+    this.scheme = scheme
+    this.$axios = axios
```

Comment:
$scheme is now scheme :D Why not doing same for axios?

Reply:
Renamed in commit 91a2fd03